### ISSUE TITLE
Data example for source fields

### ISSUE THREAD

**Issue by Shifter2600** [nyurik/kibana-vega-vis#27]
I've been struggling to get Violin Plot working. Is this the correct data query? I am only wanting to pull in the 2 fields. 
`"data": [
    {
      "name": "iris",
      "url": {
        "index": "logstash-hvms-ub-*",
        "body": {
          "_source": {"includes": ["physician", "elapse"]}
        }
      }
    }
  ],`

**Comment by Shifter2600** [comment]
also should I be using physician.keyword and if so what is the correct format?
is it "[physician.keyword]"

**Comment by Shifter2600** [comment]
so I started investigating the use of 
`"format": {
    "type": "json",
     "property": "aggregations.key.buckets"
    }`
The issue I have now is after buckets it's .0, .1, .2, .3, .........Which now eliminates the ability to get the values and keys in the same place. Also the values are in value.value.
Here is my data
`  "data": [
    {
      "name": "table",
      "url": {
        "index": "logstash-hvms-ub-*",
        "body": {
            "_source": ["physician", "elapse"],
            "aggs": {
            "key": {
              "terms": {
                "field": "physician.keyword",
                "size": 100
              },
              "aggs": {
                "value": {
                  "avg": {
                    "field": "elapse"
                  }
                }
              }
            }
          }
          }
        },
    "format": {
    "type": "json",
     "property": "aggregations.key.buckets"
    }
    }
  ],`

Here is some sample JSON.
`{
  "aggregations": {
    "key": {
      "doc_count_error_upper_bound": 0,
      "sum_other_doc_count": 0,
      "buckets": [
        {
          "key": "Dr. Rhonda Ross",
          "doc_count": 196602,
          "value": {
            "value": 2.389609464807072
          }
        },
        {
          "key": "Dr. Courtney Wright",
          "doc_count": 159187,
          "value": {
            "value": 1.0917223140080534
          }
        },
        {
          "key": "Dr. Eleanor Jones",
          "doc_count": 134339,
          "value": {
            "value": 2.4374902299406727
          }
        },
        {
          "key": "Dr. Sylvia Ortega",
          "doc_count": 127621,
          "value": {
            "value": 4.112081867404267
          }
        },
        {
          "key": "Dr. Judy Whittaker",
          "doc_count": 106158,
          "value": {
            "value": 3.8356506339607
          }
        }
      ]
    }
  }
}`

I did take a look at your solution here https://github.com/nyurik/kibana-vega-vis/issues/16 but it was too hard to follow since I am not doing any geopoints here or histogram dates.  I think if you can help me with this solution I sould be able to use this same structure on many other vega visuals.

**Comment by Shifter2600** [comment]
I believe I figured it out. I will add my Data section for an example for those instances where a user is not needing Date Time Histogram.

**Comment by nyurik** [comment]
@Shifter2600 my apologies for not replying earlier. There is a pending change to VegaLite that should make field accessing much easier, I will write a few blog posts on its use as soon as they release it in the next few days.

**Comment by nyurik** [comment]
@Shifter2600 update - [v0.4.0](https://github.com/nyurik/kibana-vega-vis/releases/tag/v0.4.0) has just been released, and in the release notes there is a link on how to access source fields from VegaLite.
